tar/tarsnap.1-man.in

```diff
@@ -397,6 +397,10 @@ Allow descent into synthetic filesystems such as procfs.
 Normally archiving of such filesystems is a silly thing to do, hence the
 name of the option.
 .TP
+\fB\--iso-dates\fP
+(-t mode only)
+Prints file and directory dates as yyyy-mm-dd hh:mm:ss.
```

Comment:
No, I mean, check that I was consistent with the sections *I've* written.  Because it's entirely possible you picked up the incorrect style after looking at somewhere I had written incorrectly.